### ISSUE TITLE
[Screen Reader-Help-Getting Started] Screen reader does not provide the information about the Confirmation popup.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2542]
Pre-Requisite: Turn on Narrator

**User Experience:**
Users who depends on Screen reader will not be able to access any page if Screen reader will not provide the information about the page opened.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Help control lying in the header section on the page and select it.
4. Navigate to 'Getting started' control and select it.
5. Navigate to various controls lying on the pane opened.
6. Verify that Screen Reader behavior.

**Actual Result:-**
Screen reader does not provide the information about the Confirmation Popup when ‘Getting Started’ is selected.

**Expected Result:-**
Screen reader should provide the information about the Confirmation Popup when ‘Getting Started’ is selected.

**Note:**
1. Same issue exists with NVDA + FF. 
2. Same Issue exists on Same screen as after select OK button navigate to Next button --> then also screen reader is not providing any information about the messages displayed on screen.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Suggested Fix:**
All the elements should be programmatically aligned correctly  
 Paragraph elements should be tagged with <p> tag 
 Link elements should have <a href> 
 Buttons should have <a> or  <a role=”button”> 
 Only interactive elements should be aligned with <a> tag, static elements on the page should not be made interactive by using <a> tag  
Reference: 
https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=111%2C211%2C131#content-structure-separation-programmatic 

Please refer the attachment
[MAS1.3.1_Help_Get started.zip](https://github.com/Microsoft/pxt/files/1179552/MAS1.3.1_Help_Get.started.zip)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
